_config.yml

```
@@ -9,9 +9,9 @@ minima:
   date_format: "%b %-d, %Y"
 
 partners:
-  - logo_url: https://www.esciencecenter.nl/img/pressroom/ESCIENCE_logo_C_nl_cyanblack.jpg
+  - logo_url: https://ewatercycle.github.io/assets/logoescience.png
```

Running locally does not show logo, for portability better to use just path
```suggestion
  - logo_url: /assets/logoescience.png
```